pkg/certificate/certificate.go

```diff
@@ -91,3 +96,47 @@ func NewFromPEM(pemCert pem.Certificate, pemKey pem.PrivateKey) (*Certificate, e
 		Expiration:   x509Cert.NotAfter,
 	}, nil
 }
+
+// CreateValidCertAndKey creates a non-expiring PEM certificate and private key
+func CreateValidCertAndKey(notBefore, notAfter time.Time) (pem.Certificate, pem.PrivateKey, error) {
```

Comment:
Since this is a test helper function, can we move it to the test file where it is used?